docs/en/workshop/a-molecular-unfolding/permissions.json

```diff
@@ -24,10 +25,9 @@
             ],
             "Resource": [
                 "arn:aws:iam::*:policy/qc*",
-                "arn:aws:iam::*:policy/qr*",
+                "arn:aws:iam::*:policy/QC*",
```

Comment:
it's better narrowing down the scope with more precise name

Reply:
also improve it in other occurrences

Reply:
it depends on the statck name and quicksight service role name.  use qc*, QC*. user can name their stack and quicksight role name more freedom.

Reply:
Not sure get it or not. any example for it?

Reply:
when user deploy the statck, if they use stack name 'QCEDD', all role/polcy names created by CF are `QCEDD*xxxxxx`/`qcedd*xxxxxx`.  In quicksight setup,  if use following the doc, a role 'qcedd-quicksight-service-role', this role will be edit in the deployment. 

here add constriant to role name qc*, QC*,  so user can name the stack to 'qcxxxx' or name quicksight role name 'qcxxxx'.

Reply:
1. why not consolidating all name to `QCEDD*` or `qcedd*`?
2. `QC` looks like is too open,  `QCEDD` is better
3. Pls highlight the limitation of default `permission.json`, if they changing the stack name, role name and policy name not staring with `QCEDD`.

Reply:
changed to  'QCEDD*', add doc to deplyment, stack name should start with 'QCEDD'